src/main/scala/com/microsoft/hyperspace/index/sources/delta/DeltaLakeRelationMetadata.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (2020) The Hyperspace Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.microsoft.hyperspace.index.sources.delta
+
+import com.microsoft.hyperspace.index.{IndexConstants, Relation}
+import com.microsoft.hyperspace.index.sources.FileBasedRelationMetadata
+
+class DeltaLakeRelationMetadata(metadata: Relation) extends FileBasedRelationMetadata {
+
+  override def refresh(): Relation = {
+    metadata.copy(options = metadata.options - "versionAsOf" - "timestampAsOf")
+  }
+
+  override def internalFileFormatName(): String = {
+    "parquet"
+  }
+
+  /**
+   * Returns enriched index properties.
+   *
+   * Delta Lake source provider adds:
+   * 1) DELTA_VERSION_HISTORY_PROPERTY logs the history of INDEX_VERSION:DELTA_TABLE_VERSION
+   *    values for each index creation & refresh.
+   *
+   * @param properties Index properties to enrich.
+   */
```

Comment:
is `@return` left out intentionally?

Reply:
Done. By the way Javadoc and Scaladoc have a conflicting guideline on this: Javadoc says you shouldn't omit `@return` even if it repeats the method description; Scaladoc says otherwise.

Reply:
OK. I will let you drive the effort to make comments consistent. You can pick either style as long as they are consistent in the repo.